app/shared/ui-foundation/src/desktopMain/kotlin/platform/window/WindowsWindowUtils.kt

```diff
@@ -81,15 +143,49 @@ class WindowsWindowUtils : AwtWindowUtils() {
         }.getOrElse { false }
     }
 
-    fun hideTitleBar(hwnd: HWND?) {
-        var style = User32.INSTANCE.GetWindowLong(hwnd, WinUser.GWL_STYLE).toLong()
-        style = style and WinUser.WS_CAPTION.toLong().inv() and WinUser.WS_BORDER.toLong()
-            .inv() and WinUser.WS_THICKFRAME.toLong().inv()
-        User32.INSTANCE.SetWindowLong(hwnd, WinUser.GWL_STYLE, style.toInt())
-        User32.INSTANCE.SetWindowPos(
-            hwnd, null, 0, 0, 0, 0,
-            WinUser.SWP_NOMOVE or WinUser.SWP_NOSIZE or SWP_NOZORDER or SWP_FRAMECHANGED,
-        )
+    fun extendToTitleBar(
+        platformWindow: PlatformWindow,
+        windowScope: WindowScope? = platformWindow.windowScope,
+        hitTest: (Float, Float) -> Int
+    ) {
+        if (windowScope != null) {
+            platformWindow.titleBarWindowProc?.dispose()
+            platformWindow.titleBarWindowProc = TitleBarWindowProc(
+                windowScope.window,
+                extendedUser32,
+                dwmAPi,
+                hitTest,
+            )
+        }
+    }
+
+    fun windowIsActive(platformWindow: PlatformWindow): Flow<Boolean?> {
+        return snapshotFlow { platformWindow.titleBarWindowProc }
+            .flatMapConcat {
+                it?.windowIsActive ?: flow<Boolean?> { emit(null) }
```

Comment:
default to `MutableStateFlow<Boolean?>(false)`

Reply:
MSF changes semantics.

use `flowOf(null)` instaed